pkg/plugins/resources/file/source.go

```diff
@@ -27,6 +28,12 @@ func (f *File) Source(workingDir string) (string, error) {
 		return "", fmt.Errorf("Validation error: the provided manifest configuration had the following validation errors:\n%s", strings.Join(validationErrors, "\n\n"))
 	}
 
+        # Relative path is used when an SCM is associated with the file resource: means the file is on a remote SCM (hence relative path)
```

Comment:
```suggestion
        // Relative path is used when an SCM is associated with the file resource: means the file is on a remote SCM (hence relative path)
```

My bad, E_TOO_MUCH_GROOVY